crates/rune-cli/src/main.rs

```diff
@@ -622,6 +628,14 @@ async fn run_path(args: &Args, options: &Options, path: &Path) -> Result<ExitCod
                     }
                 }
 
+                if constants.peek().is_some() {
```

Comment:
Should check `dump_constants`?

```suggestion
                if runargs.dump_constants && constants.peek().is_some() {
```

Reply:
Good point